src/bulk-load.ts

```diff
@@ -41,12 +41,19 @@ type InternalOptions = {
   lockTable: boolean,
 };
 
-export interface Options {
+export type Options = {
```

Comment:
looks like 'interface' is more flexible compared to 'type' in many cases. I will work on modifying to use 'interface' instead of 'type' wherever possible.